Podfile

```diff
@@ -125,8 +125,11 @@ end
 
 post_install do |installer|
   installer.pods_project.targets.each do |target|
+    usePackageSwift = %w[ Eureka ]
     target.build_configurations.each do |config|
-      config.build_settings['SWIFT_VERSION'] = '3.0'
+      unless usePackageSwift.include? target.name
+        config.build_settings['SWIFT_VERSION'] = '3.0'
+      end
```

Comment:
I like this approach. I figured we were going to need to do this as we migrated anyway, so it's good to see it look nice in practice.